x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/actions/migrations.ts

```diff
@@ -0,0 +1,451 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { SECURITY_SOLUTION_SAVED_OBJECT_INDEX } from '@kbn/core-saved-objects-server';
+
+import { DETECTION_ENGINE_RULES_URL } from '@kbn/security-solution-plugin/common/constants';
+import {
+  getLegacyActionSOById,
+  getLegacyActionNotificationSOById,
+  getRuleSOById,
+} from '../../../../../detection_engine_api_integration/utils';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+/**
+ * @deprecated Once the legacy notification system is removed, remove this test too.
+ */
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const esArchiver = getService('esArchiver');
+
+  describe('@ess @skipInQA actions migrations', () => {
```

Comment:
If we add `@ess` - should that be enough to exclude all serverless related tags since `@skipInQA` is particular to serverless?

Reply:
Note: being addressed in follow up PR to unblock others.